### PR TITLE
Update l10nzhtw.properties

### PR DESCRIPTION
**Pull request by JE-Chen**
Add 1.1 version info
[SonarCloud](https://sonarcloud.io/project/overview?id=JE-Chen_sonar-l10n-zh-tw)

### CODE REVIEW

**Comment by ganncamp** [comment]
Have you communicated with @timlee about this to get his agreement for your takeover? Or are you just taking up the mantel because you believe the project is abandoned?

**Comment by JE-Chen** [comment]
> Have you communicated with @timlee about this to get his agreement for your takeover? Or are you just taking up the mantel because you believe the project is abandoned?

Yes, I have contacted Tim Lee. He is my teacher.

**Comment by ganncamp** [comment]
Thanks for the confirmation @timlee! I'm happy to move forward with this.

File: l10nzhtw.properties [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+homepageUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw
+archivedVersions=1.0
+publicVersions=1.0, 1.1
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
-defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+defaults.mavenArtifactId=sonar-l10n-zh-tw-plugin
+
+1.1.description=SonarQube Traditional Chinese Pack Support SonarQube 10.7
+1.1.sqVersions=[10.7,LATEST]
```
[the remark this answers is not in the document]
Comment (JE-Chen) [reply l10nzhtw.properties:11]: Should I also make translations for versions 10.0 - 10.6? How do I test BundlesUpToDate when I add translations for older versions?

[the remark this answers is not in the document]
Comment (ganncamp) [reply l10nzhtw.properties:11]: TBH, I don't know what BundlesUpToDate is.
You _could_ make translations for the intermediate versions. But I wasn't trying to burden you with extra work. I was hinting that this 10.7 version be set to start at 10.0. Even if it's imperfect/gappy, it's better than nothing.

[the remark this answers is not in the document]
Comment (JE-Chen) [reply l10nzhtw.properties:11]: Like this?
sqVersions=[10.0,10.0.*,10.1,10.1.*,10.2,10.2.*,10.3,10.3.*, .... -10.7,LATEST]
or can I simply do this?
sqVersions=[10.*,10.*.*,LATEST]

**Review by ganncamp — CHANGES_REQUESTED** [review]

**Review by ganncamp — APPROVED** [review]

**Comment by ganncamp** [comment]
FYI, this is approved, but I can't merge until the build succeeds and I'm not finding the failure reason in the logs, so I don't know (yet) how to proceed. I've asked for help & will come back to this tomorrow.

**Comment by ganncamp** [comment]
I haven't forgotten this. I'm in line for the help I requested internally.

File: l10nzhtw.properties [hunk 2]
```diff
@@ -1,14 +1,20 @@
 category=Localization
 description=SonarQube Traditional Chinese Language Pack (not compatible with the "Chinese Pack" which uses Simplified Chinese)
-homepageUrl=https://github.com/timlee/sonar-l10n-zh-tw
+homepageUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw
 archivedVersions=
-publicVersions=1.0
+publicVersions=1.0, 1.1
```
Comment (ganncamp) [inline l10nzhtw.properties:5]: Can you remove the space here, please

**Review by ganncamp — CHANGES_REQUESTED** [review]

**Review by ganncamp — APPROVED** [review]

**Comment by ganncamp** [comment]
I have to roll this back because the metadata generation job is failing with this error:
```
[ERROR] Failed to execute goal org.sonarsource.update-center:sonar-update-center-mojo:1.32.0.2441:generate-metadata (default-cli) on project sonar-update-center-properties: Fail to execute mojo: Plugin sonar-l10n-zh-tw-plugin.jar is declared with key 'null' in its MANIFEST, but with key 'l10nzhtw' in the update center -> [Help 1]
```
And when I look at the manifest, it has only
```
Manifest-Version: 1.0
Main-Class: Main
```
Versus a much more robust manifest in the previous version's jar. Whatever changes you made to the pom, you should undo them. 
And now that I look, I see a lot more in your jar than there should be, and I also notice that your jar name doesn't have a version number in it. That may be fine for now, but what happens when you put out the next version? They'll clash. Your jar names need to be unique.

So for now, I'm going to roll this back entirely.

**Comment by JE-Chen** [comment]
> I have to roll this back because the metadata generation job is failing with this error:
> 
> ```
> [ERROR] Failed to execute goal org.sonarsource.update-center:sonar-update-center-mojo:1.32.0.2441:generate-metadata (default-cli) on project sonar-update-center-properties: Fail to execute mojo: Plugin sonar-l10n-zh-tw-plugin.jar is declared with key 'null' in its MANIFEST, but with key 'l10nzhtw' in the update center -> [Help 1]
> ```
> 
> And when I look at the manifest, it has only
> 
> ```
> Manifest-Version: 1.0
> Main-Class: Main
> ```
> 
> Versus a much more robust manifest in the previous version's jar. Whatever changes you made to the pom, you should undo them. And now that I look, I see a lot more in your jar than there should be, and I also notice that your jar name doesn't have a version number in it. That may be fine for now, but what happens when you put out the next version? They'll clash. Your jar names need to be unique.
> 
> So for now, I'm going to roll this back entirely.

I already updated the jar file in the same release.

**Comment by ganncamp** [comment]
I'll need a new PR

**Pull request merged**